nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -8,31 +8,33 @@ tags:
 - Configuration
 ---
 
-The behavior of each persister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing available properties, or using code API. The exact same settings are exposed via `app.config` file and code configuration API.
+In NServiceBus.Azure the behavior of the AzureStoragePersister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing the available properties, or by using the code API. The exact same settings are exposed via the `app.config` and the code configuration API.
+
+In NServiceBus.Persistence.AzureStorage v1 and above XML based configuration is no longer available. Configuring the behavior of the persister is done using the code configuration API.
```

Comment:
should be "Version 1 and above"

See https://github.com/particular/docs.particular.net#version-language